sample_code/redirectQR_sample.ts

```diff
@@ -0,0 +1,60 @@
+import "https://deno.land/std@0.209.0/dotenv/load.ts";
+import { Client } from "../src/mod.ts";
```

Comment:
Should we standardize on local version or do we need to first use the local, then update the dependency later to the deno.land url later?

Reply:
Oooooops!!! That should be deno.land. Will fix!